jinja2_standalone_compiler/__init__.py

```diff
@@ -46,12 +49,14 @@ def main(path, settings=None):
             continue
 
         html_template, _ = os.path.splitext(jinja_template)
-        html_template = '{}.html'.format(html_template)
+        if output_options['remove_double_extension']:
```

Comment:
I didn't try this, but I guess this is unsafe and may raise a `KeyError`. What about `output_options.get('remove_double_extension')`?